org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/xmleditor/AbstractXmlRuntimeModule.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * generated by Xtext
+ */
+package org.eclipse.xtext.ui.tests.xmleditor;
+
+import com.google.inject.Binder;
+import com.google.inject.Provider;
+import com.google.inject.name.Names;
+import java.util.Properties;
+import org.eclipse.xtext.Constants;
+import org.eclipse.xtext.IGrammarAccess;
+import org.eclipse.xtext.naming.DefaultDeclarativeQualifiedNameProvider;
+import org.eclipse.xtext.naming.IQualifiedNameProvider;
+import org.eclipse.xtext.parser.IParser;
+import org.eclipse.xtext.parser.ITokenToStringConverter;
+import org.eclipse.xtext.parser.antlr.AntlrTokenDefProvider;
+import org.eclipse.xtext.parser.antlr.AntlrTokenToStringConverter;
+import org.eclipse.xtext.parser.antlr.IAntlrTokenFileProvider;
+import org.eclipse.xtext.parser.antlr.ITokenDefProvider;
+import org.eclipse.xtext.parser.antlr.Lexer;
+import org.eclipse.xtext.parser.antlr.LexerProvider;
+import org.eclipse.xtext.resource.IContainer;
+import org.eclipse.xtext.resource.IResourceDescriptions;
+import org.eclipse.xtext.resource.containers.IAllContainersState;
+import org.eclipse.xtext.resource.containers.ResourceSetBasedAllContainersStateProvider;
+import org.eclipse.xtext.resource.containers.StateBasedContainerManager;
+import org.eclipse.xtext.resource.impl.ResourceDescriptionsProvider;
+import org.eclipse.xtext.resource.impl.ResourceSetBasedResourceDescriptions;
+import org.eclipse.xtext.scoping.IGlobalScopeProvider;
+import org.eclipse.xtext.scoping.IScopeProvider;
+import org.eclipse.xtext.scoping.IgnoreCaseLinking;
+import org.eclipse.xtext.scoping.impl.AbstractDeclarativeScopeProvider;
+import org.eclipse.xtext.scoping.impl.DefaultGlobalScopeProvider;
+import org.eclipse.xtext.scoping.impl.ImportedNamespaceAwareLocalScopeProvider;
+import org.eclipse.xtext.serializer.ISerializer;
+import org.eclipse.xtext.serializer.impl.Serializer;
+import org.eclipse.xtext.serializer.sequencer.ISemanticSequencer;
+import org.eclipse.xtext.serializer.sequencer.ISyntacticSequencer;
+import org.eclipse.xtext.service.DefaultRuntimeModule;
+import org.eclipse.xtext.service.SingletonBinding;
+import org.eclipse.xtext.ui.tests.xmleditor.parser.antlr.XmlAntlrTokenFileProvider;
+import org.eclipse.xtext.ui.tests.xmleditor.parser.antlr.XmlParser;
+import org.eclipse.xtext.ui.tests.xmleditor.parser.antlr.lexer.InternalXmlLexer;
+import org.eclipse.xtext.ui.tests.xmleditor.scoping.XmlScopeProvider;
+import org.eclipse.xtext.ui.tests.xmleditor.serializer.XmlSemanticSequencer;
+import org.eclipse.xtext.ui.tests.xmleditor.serializer.XmlSyntacticSequencer;
+import org.eclipse.xtext.ui.tests.xmleditor.services.XmlGrammarAccess;
+import org.eclipse.xtext.ui.tests.xmleditor.validation.XmlValidator;
+
+/**
+ * Manual modifications go to {@link XmlRuntimeModule}.
+ */
+@SuppressWarnings("all")
+public abstract class AbstractXmlRuntimeModule extends DefaultRuntimeModule {
```

Comment:
Personally I'd prefer XmlTestLanguage, e.g. a name that makes it obvious that this is just for testing purpose. Maybe even prefix it with the bug id.